components/AddTestimony/AddTestimony.js

```diff
@@ -18,12 +18,18 @@ const AddTestimony = (props) => {
       
       <Modal show={showAddComment} onHide={handleCloseAddComment} size="lg">
         <Modal.Header closeButton onClick={handleCloseAddComment}>
-            <Modal.Title>{bill ? bill.billNumber + " - " + bill.title : ""}</Modal.Title>
+            <Modal.Title>{bill ? bill.BillNumber + " - " + bill.Title : ""}</Modal.Title>
         </Modal.Header>
         <Modal.Body>
           <>
             <div className="text-center">
-              <textarea className="form-control" rows="20" placeholder="My comments on this bill" required></textarea>
+              <select className="form-control">
```

Comment:
I tried this change, but it removed the free text area where testifiers could put in their comments.
![image](https://user-images.githubusercontent.com/67333843/153762905-3a6143e6-3c55-4885-805a-470c5020850a.png)

Reply:
That is a good idea.  I made a change so that nothing is selected by default.
![image](https://user-images.githubusercontent.com/67333843/153765574-39589717-bdde-41d2-a23f-c3449597e4f7.png)